integration-tests/deployment/devenv/chain.go

```diff
@@ -0,0 +1,125 @@
+package devenv
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/sethvargo/go-retry"
+	chainselectors "github.com/smartcontractkit/chain-selectors"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+)
+
+type ChainConfig struct {
+	ChainId         uint64
+	ChainName       string
+	ChainType       string
+	WsRpcs          []string
+	HttpRpcs        []string
+	PrivateWsRpcs   []string // applicable for private chains only so that nodes within same cluster can connect internally
+	PrivateHttpRpcs []string // applicable for private chains only so that nodes within same cluster can connect internally
```

Comment:
Style:

```suggestion
	WSRPCs          []string
	HTTPRPCs        []string
	PrivateWSRPCs   []string // applicable for private chains only so that nodes within same cluster can connect internally
	PrivateHTTPRPCs []string // applicable for private chains only so that nodes within same cluster can connect internally
```

Lets try to follow the style guide where possible. https://google.github.io/styleguide/go/decisions#initialisms